dask_geopandas/core.py

```diff
@@ -12,10 +12,9 @@
 from shapely.geometry import box
 
 
-def _set_crs(df, crs):
+def _set_crs(df, crs, allow_override):
     """Return a new object with crs set to ``crs``"""
-    df = df.copy(deep=False)
-    df.crs = crs
+    df = df.copy(deep=False).set_crs(crs, allow_override=allow_override)
```

Comment:
Since `set_crs` doesn't work in place by default, do we need `copy(deep=False)` here?

Reply:
I don't think we do, I removed it